src/OagBundle/Resources/views/Activity/enhance.html.twig

```diff
@@ -2,14 +2,34 @@
 
 {% block title %}OagBundle:Activity:enhance{% endblock %}
 
+{% block stylesheets %}
+    {{ parent() }}
+    <link href="{{ asset('bundles/oag/neonmap/dist/neonmap.css') }}" rel="stylesheet" />
+{% endblock %}
+
 {% block h2 %}{{ activity.name }} [{{ activity.id }}]{% endblock %}
 
 {% block body %}
-    {{ dump(activity)  }}
 
+    <div id="existing-locations">
+        <h3>Existing locations</h3>
+        <div id="existing-map-block"
+             data-geojson='{{ mapdata }}'
+             ></div>
+    </div>
     {{ form(form) }}
 {% endblock %}
 
+{% block javascripts %}
+    {{ parent() }}
+    <script src="{{ asset('bundles/oag/neonmap/dist/neonmap.js') }}"></script>
+    <script>
+        $( document ).ready(function() {
+            $('#existing-map-block').neonmap({image_path: '../../../../bundles/oag/neonmap/dist/images'});
```

Comment:
maybe we could use 
`'{{ asset('bundles/oag/neonmap/dist/images') }}'` for the path here?